public/js/admin.js

```diff
@@ -47,7 +47,8 @@ var displayAccountsTable = function() {
             if (data['status'] === 401) {
                 window.location.href = '/';
             } else {
-                failSnackbar('Something went wrong, please try again later!');
+                data = data.responseJSON;
+                failSnackbar(getErrorFromResponse(data));
```

Comment:
why not just pass data.responseJSON

Reply:
we should maintain the original data object incase later usage

Reply:
well, passing data.responseJSON is not up to us, this is just how an ajax request works. I agree about not changing the original data, will fix that